terraso_backend/apps/core/models/landscapes.py

```diff
@@ -40,6 +43,26 @@ class Landscape(SlugModel):
     )
     groups = models.ManyToManyField(Group, through="LandscapeGroup")
 
+    area_types = models.JSONField(blank=True, null=True)
```

Comment:
I think I rebased onto staging to make sure that I had the most recent changes for testing. This is a good point. It might make sense to merge this after the profile landscape changes are merged to main.